docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -18,77 +17,56 @@
 
 
 
-Model Optimizer is a cross-platform command-line tool that facilitates the transition between training and deployment environments, performs static model analysis, and adjusts deep learning models for optimal execution on end-point target devices.
-
-To use it, you need a pre-trained deep learning model in one of the supported formats: TensorFlow, PyTorch, PaddlePaddle, TensorFlow Lite, MXNet, Caffe, Kaldi, or ONNX. Model Optimizer converts the model to the OpenVINO Intermediate Representation format (IR), which you can infer later with :doc:`OpenVINO™ Runtime <openvino_docs_OV_UG_OV_Runtime_User_Guide>`.
-
-Note that Model Optimizer does not infer models.
-
-The figure below illustrates the typical workflow for deploying a trained deep learning model:
-
-.. image:: _static/images/BASIC_FLOW_MO_simplified.svg
-
-where IR is a pair of files describing the model:
-
-* ``.xml`` - Describes the network topology.
-
-* ``.bin`` - Contains the weights and biases binary data.
-
-The OpenVINO IR can be additionally optimized for inference by :doc:`Post-training optimization <pot_introduction>` that applies post-training quantization methods.
-
-How to Run Model Optimizer
-##########################
-
-To convert a model to IR, you can run Model Optimizer by using the following command:
+To convert a model to IR, you can run model conversion API by using the following command:
 
 .. code-block:: sh
 
    mo --input_model INPUT_MODEL
 
 
-If the out-of-the-box conversion (only the ``--input_model`` parameter is specified) is not successful, use the parameters mentioned below to override input shapes and cut the model:
+If the out-of-the-box conversion (only the ``input_model`` parameter is specified) is not successful, use the parameters mentioned below to override input shapes and cut the model:
```

Comment:
Just previous example is still in CLI constasting with this Python API shift. They are not aligned. Please provide a Python example above. Also please reflect the fact that `input_model` argument is optional. So the first unnamed argument will be used as input_model making both `convert_model(input_model=mode)` and `convert_model(model)` having the same effect. This is unfortunately different for `mo` called as CLI and `convert_model`, so CLI `mo` alsways requires `--input_model`. CCed @popovaan.

Reply:
It's been addressed with the [3d74c34](https://github.com/openvinotoolkit/openvino/pull/17454/commits/3d74c34c3cbb175693eb296bc60e6dbd3cded)d68 commit.